code/messaging/build.gradle

```diff
@@ -161,8 +178,10 @@ signing {
 
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
+    //implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
+    //compileOnly files('libs/android-core-library-phone-coreBundle-debug.aar')
+    //compileOnly files('libs/optimize-phone-debug.aar')
+    implementation project(":messaging:aar")
```

Comment:
this can be removed once the optimize extension and the android core changes supporting the messaging extension are available via maven.

Reply:
currently the two dependencies are provided via aar files.